src/CLI/Context/AddTaskBuffer.cpp

```diff
@@ -0,0 +1,50 @@
+//
+// Created by rodion on 9/9/20.
+//
+
+#include "AddTaskBuffer.h"
+
+AddTaskBuffer::AddTaskBuffer() {
+  this->clearBuffer();
+}
+
+void AddTaskBuffer::clearBuffer() {
+  this->name_ = "";
+  this->label_ = "";
+  this->priority_ = Priority::EMPTY;
+  this->date_ = boost::gregorian::date{boost::gregorian::not_a_date_time};
+  this->parent_id_ = 0;
+  this->is_subtask_ = false;
```

Comment:
if parent id exist - this is already subtask. you can remove "is_subtask" variable.